packages/core/src/test/awsService/appBuilder/samProject.test.ts

```diff
@@ -78,7 +78,7 @@ describe('samProject', () => {
             // simulate error when no samconfig.toml file in directory
             const result = await getStackName(projectRoot)
             assert.deepStrictEqual(result, {})
-            assertLogsContain('No stack name or region information available in samconfig.toml', true, 'info')
+            assertLogsContain('No stack name or region information available in samconfig.toml: %O', true, 'info')
```

Comment:
Yes, but believe it is only because we are in the `TestLogger`, which adds the raw strings to an array. (https://github.com/aws/aws-toolkit-vscode/blob/b2ebc10342f63ac95cd3055393e715cb369ec450/packages/core/src/test/testLogger.ts)